core-client/config/client_local_threshold.toml

```diff
@@ -23,3 +17,32 @@ decryption_mode = "NoiseFloodSmall"
 
 # fhe_params = "Test" # Small, insecure parameters for testing
 fhe_params = "Default" # Large, secure parameters
+
+# List of connection info to each of the cores
+[[cores]]
+# The ID of the given KMS server (monotonically increasing positive integer starting at 1)
+party_id = 1
+# The address of the given KMS server, including the port
+address = "localhost:50100"
+# The S3 endpoint where the public material of the given server can be reached
+s3_endpoint = "http://localhost:9000/kms"
+# The folder at the S3 endpoint where the data is stored
+object_folder = "PUB-p1"
+
+[[cores]]
```

Comment:
It is _not_ recommended to repeat a table header in TOML, see spec: https://toml.io/en/v1.0.0#table.
I think there should only be a single `[cores]` header. Also why the double `[[`?